src/main/webapp/resources/css/makeRentPost.css

```diff
@@ -0,0 +1,41 @@
+#map {
```

Comment:
Use alphabetize declarations of tags throughout the file. Also for the properties inside css tags.